cpp/translation_db_tool.cc

```diff
@@ -99,6 +116,20 @@ void GetPossibleCategories(DbConnection * const connection, std::set<std::string
 }
 
 
+bool IsValid3LetterLanguageCode(const std::string &language_code_candidate) {
+    static std::vector<std::string> VALID_LANGUAGE_CODES{
+        "deu", "eng", "fra"
+    };
+
+    for (const auto &valid_language_code : VALID_LANGUAGE_CODES) {
```

Comment:
Isn't this the same like `return std::find(VALID_LANGUAGE_CODES.cbegin(), VALID_LANGUAGE_CODES.cend(), language_code_candidate) != VALID_LANGUAGE_CODES.end()`?